pkg/util/util.go

```diff
@@ -156,6 +158,30 @@ func TruncateString(str string, maxLen int) string {
 	return str
 }
 
+// GetAbsPath returns absolute path from passed file path resolving even ~ to user home dir and any other such symbols that are only
+// shell expanded can also be handled here
+func GetAbsPath(path string) (string, error) {
+	// Only shell resolves `~` to home so handle it specially
+	if strings.HasPrefix(path, "~") {
+		usr, err := user.Current()
+		if err != nil {
+			return path, errors.Wrapf(err, "unable to generate a random name while resolving %s to absolute path", path)
```

Comment:
That doesn't look correct.

Reply:
:laughing: 
yeah my bad...
will fix it

Reply:
fixed it